lib/shipit/stack_commands.rb

```diff
@@ -106,7 +106,7 @@ def git_cmd_succeeds?(path)
         .success?
     end
 
-    def git_clone(url, path, branch: 'master', **kwargs)
+    def git_clone(url, path, branch: 'main', **kwargs)
```

Comment:
I suspect that this change is fine because `branch` is a required field when creating a stack. I don't know that there's a real situation where you would end up here and `git_clone` would be nil.